src/plasma/plasma_manager.cc

```diff
@@ -1311,33 +1312,35 @@ void log_object_hash_mismatch_error_object_callback(ObjectID object_id,
 }
 
 void process_add_object_notification(PlasmaManagerState *state,
-                                     ObjectInfo object_info) {
-  ObjectID obj_id = object_info.obj_id;
+                                     ObjectID object_id,
```

Comment:
Why do you pass in the arguments here individually instead of passing in the `ObjectInfo` object? Just curious.